modules/@apostrophecms/notification/index.js

```diff
@@ -218,37 +244,43 @@ module.exports = {
         before: '@apostrophecms/global',
         middleware: async (req, res, next) => {
           let start;
-          let displayingIds;
+          let modifiedOnOrSince;
+          let seenIds;
           try {
-            if (req.method.toUpperCase() !== 'GET' || req.url !== self.action) {
+            const reqUrl = new URL(req.url, req.baseUrl);
+            if (req.method.toUpperCase() !== 'GET' || reqUrl.pathname !== self.action) {
               return next();
             }
             if (!(req.user && req.user._id)) {
               throw self.apos.error('invalid');
             }
             start = Date.now();
-            displayingIds = self.apos.launder.ids(req.body.displayingIds);
+            modifiedOnOrSince = req.query.modifiedOnOrSince && self.apos.launder.date(req.query.modifiedOnOrSince);
+            seenIds = req.query._id && self.apos.launder.ids(req.query._id) && req.query._id.split(',_id=');
```

Comment:
This works, but please use `seenIds` for the query parameter too, to make it clear what it does.

Reply:
Done